lib/game/configureGamePass.js

```diff
@@ -0,0 +1,100 @@
+// Includes
+const http = require('noblox.js/lib/util/http').func
+const getGeneralToken = require('noblox.js/lib/util/getGeneralToken').func
+
+// Args
+exports.required = ['id', 'name']
+exports.optional = ['description', 'price', 'jar']
```

Comment:
You can include image changes: https://github.com/suufi/noblox.js/blob/master/lib/asset/uploadItem.js
I don't think it's that different from uploadItem